Frameworks/Starboard/EbrFile.cpp

```diff
@@ -189,27 +234,27 @@ bool EbrGetRootMapping(const char* dirName, char* dirOut, uint32_t maxLen) {
         return true;
     }
     if (_stricmp(dirName, "Cache") == 0) {
-        sprintf_s(dirOut, maxLen, "%s\\cache", g_WritableFolder);
+        sprintf_s(dirOut, maxLen, "%ls\\cache", _EbrGetBaseWriteablePath());
         mkdir(dirOut);
         return true;
     }
     if (_stricmp(dirName, "Library") == 0) {
-        sprintf_s(dirOut, maxLen, "%s\\Library", g_WritableFolder);
+        sprintf_s(dirOut, maxLen, "%ls\\Library", _EbrGetBaseWriteablePath());
```

Comment:
I know it's not your code, but interesting write. 

---
In reply to: [98273229](https://github.com/Microsoft/WinObjC/pull/1849#discussion_r98273229) [](ancestors = 98273229)